src/app/settings.py

```diff
@@ -143,3 +143,10 @@
 
 AUTH_USER_MODEL = "users.User"
 
+# Sentry
+
+if not DEBUG:
```

Comment:
I think the lab env should have sentry, so we have to change this if I guess.

Reply:
Maybe we can filter by the `ENV` env var then, since localhost usually has `DEBUG` set to `True` and we don't want Sentry in that case, I guess.

Reply:
Yeah, but lab env has DEBUG true as well, maybe created an `ENABLE_SENTRY` env, what do you think?

Reply:
I'm not a big fan of increasing the minimal number of env vars to start a project... but if we set a default value that might be a reasonable approach! :)